inc/package.class.php

```diff
@@ -188,8 +192,8 @@ public function prepareInputForAdd($input) {
       }
       try {
          $uploadedFile = $preparedFile['uploadedFile'];
-         $input['filename'] = $this->fields['entities_id'] . '/' . uniqid() . '_' . basename($uploadedFile);
-         $destination = FLYVEMDM_PACKAGE_PATH . '/' . $input['filename'];
+         $input['filename'] = 'flyvemdm/package/' . $this->fields['entities_id'] . '/' . uniqid() . '_' . basename($uploadedFile);
+         $destination = GLPI_DOC_DIR . '/' . $input['filename'];
```

Comment:
change how the path to files is saved. Now based on GLPI_DOC_DIR instead of a subdir. This makes process in Orion easier, and easily comaptible with other plugins (armadito)